src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/V2/DownloadedPackagesSQLLiteStoreV2.swift

```diff
@@ -0,0 +1,539 @@
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import FMDB
+import Foundation
+
+final class DownloadedPackagesSQLLiteStoreV2 {
```

Comment:
Currently the migration tests access previous Store versions. To align with the existing mechanism I copied the store and created a V2 version. A refactoring ticket to tackle this is being created.

Reply:
To review the changes please try:

`opendiff src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/V1/DownloadedPackagesSQLLiteStoreV1.swift src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/V2/DownloadedPackagesSQLLiteStoreV2.swift`

and 

`opendiff src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/V1/DownloadedPackagesStoreV1.swift src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/V2/DownloadedPackagesStoreV2.swift`